gradle/dependency-management/capabilities.json

```diff
@@ -102,5 +102,13 @@
         ],
         "selected": "cglib:cglib",
         "upgrade": "3.2.6"
+    },
+    {
+        "name": "jsch",
+        "providedBy": [
+            "com.jcraft:jsch", "com.github.mwiede:jsch"
+        ],
+        "selected": "com.github.mwiede:jsch",
+        "upgrade": "0.2.0"
```

Comment:
This doesn't work as expected because I still see `com.jcraft:jsch` in the listed dependencies. Should `capabilities.json` apply to all projects, included builds, settings files, etc...?

![image](https://user-images.githubusercontent.com/897017/162003686-aee4980b-ebfd-41ed-a397-a3716c577fa7.png)

Is there a task in the Gradle project that will list the origin of all dependencies?

Reply:
I've found an option in IntelliJ to show where a dependency is used, here's the report 

![image](https://user-images.githubusercontent.com/897017/163262557-361ad8f9-c150-48ff-8d43-3b42699d71e0.png)

I'll see if I can zap these remaining usages